codegen/sdk-codegen/build.gradle.kts

```diff
@@ -16,8 +16,10 @@ plugins {
     id("software.amazon.smithy") version "0.5.3"
 }
 
+val smithyVersion: String by project
 dependencies {
     implementation(project(":smithy-aws-swift-codegen"))
+    implementation("software.amazon.smithy:smithy-smoke-test-traits:$smithyVersion")
```

Comment:
@milesziemer Will this dependency fail our build if we don't update to the Smithy version that includes it at the same time?

(We pin to specific Smithy versions in this project and in smithy-swift)

Reply:
It looks like you're on 1.42.0 based on [gradle.properties](https://github.com/smithy-lang/smithy-swift/blob/922d0663ff94bf4895088f13c6d3afcd91c703fa/gradle.properties#L6), right? 1.42.0 has smithy-smoke-test-traits